backend/coreapp/compiler_wrapper.py

```diff
@@ -180,6 +182,7 @@ def compile_code(
                         "FUNCTION": function,
                         "MWCIncludes": "/tmp",
                         "TMPDIR": "/tmp",
+                        "QEMU_IRIX": sandbox.rewrite_path(compiler.path) + QEMU_IRIX,
```

Comment:
it's a relative path from the compiler dir, right? in that case, this doesn't need a static `QEMU_IRIX` var I don't think. the path can be encoded into the compile command.

Reply:
The reason for this variable is to avoid the qemu-irix no such file or directory error due to the sandbox (if I'm not wrong), it also simplifies the compilation command.

Reply:
I'm saying the compiler command can just say `$(pwd)/qemu-irix` or something to that effect.

https://discord.com/channels/897066363951128586/897075447534354462/1079654737911230484

Reply:
so to reiterate, we copy qemu-irix into COMPILER_DIR in download.py and then we don't need a variable to refer to where it is, since it's always in the same place. also need for `QEMU_IRIX` above.

Reply:
cc='"${COMPILER_DIR}"/usr/bin/qemu-irix -L "${COMPILER_DIR}" "${COMPILER_DIR}/usr/bin/cc" -c -Xcpluscomm -G0 -non_shared -woff 649,838,712 -32 ${COMPILER_FLAGS} -o "${OUTPUT}" "${INPUT}"',

> I'm saying the compiler command can just say `$(pwd)/qemu-irix` or something to that effect.
> 
> https://discord.com/channels/897066363951128586/897075447534354462/1079654737911230484

i think this is better
 cc='"${COMPILER_DIR}"/usr/bin/qemu-irix -L "${COMPILER_DIR}" "${COMPILER_DIR}/usr/bin/cc" -c -Xcpluscomm -G0 -non_shared -woff 649,838,712 -32 ${COMPILER_FLAGS} -o "${OUTPUT}" "${INPUT}"',